docs/conf/prague/2020/news/cfp-open.rst

```diff
@@ -0,0 +1,46 @@
+:template: {{year}}/generic.html
+
+.. post:: June 22, 2020
+   :tags: prague-2020, website, cfp, tickets
+
+Announcing Prague Call for Proposals
+====================================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/2019/cfp/>`_.
+
+Due to the circumstances that you're all familiar with, we're opening it later than usual, and for a shorter length of time.
+And we know that it hasn't been an easy year. Nevertheless we'd love to see you on our online stage in October, to share your ideas with our inspiring community of documentarians.
+
+If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, editor, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`__ on the website.
+The Call for Proposals will be open until **Midnight {{tz}} on {{cfp.ends}}**.
+
```

Comment:
We should mention something about being excited for speakers outside of Europe this year, given the online nature of the event. We should also ask for people to help promote it to folks who might have historically not been able to come.